sql/npc_list.sql

```diff
@@ -6429,7 +6429,7 @@ INSERT INTO `npc_list` VALUES (17105528,'49','     ',0,0.000,0.000,0.000,0,40,40
 INSERT INTO `npc_list` VALUES (17105529,'50','     ',0,0.000,0.000,0.000,0,40,40,0,0,0,6,27,0x0100000100100020003000400050006000700000,32,'WOTG');
 INSERT INTO `npc_list` VALUES (17105530,'51','     ',0,0.000,0.000,0.000,0,40,40,0,0,0,6,27,0x0100000100100020003000400050006000700000,32,'WOTG');
 INSERT INTO `npc_list` VALUES (17105531,'Renadile','Renadile',56,137.182,-2.000,120.107,20,40,40,0,0,0,0,27,0x0100070300108120203081402050006000700000,32,'WOTG');
-INSERT INTO `npc_list` VALUES (17105532,'Door:\"LionSpri','Door:"Lion Springs"',158,96.716,-1.000,107.262,32769,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'WOTG');
+INSERT INTO `npc_list` VALUES (17105532,'_6ep','Door:"Lion Springs"',158,96.716,-1.000,107.262,32769,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'WOTG');
```

Comment:
@Zenny89 since its not a real door that can be opened door lets just call it `Lion_Springs` and rename the lua to `Lion_Springs.lua`